[2025-02-03T03:28:46.228+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-02-03T03:28:46.261+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: csv_dag.clear_data_task manual__2025-02-03T03:28:21.469953+00:00 [queued]>
[2025-02-03T03:28:46.276+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: csv_dag.clear_data_task manual__2025-02-03T03:28:21.469953+00:00 [queued]>
[2025-02-03T03:28:46.277+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 2
[2025-02-03T03:28:46.306+0000] {taskinstance.py:2327} INFO - Executing <Task(PythonOperator): clear_data_task> on 2025-02-03 03:28:21.469953+00:00
[2025-02-03T03:28:46.318+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=1364) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-02-03T03:28:46.321+0000] {standard_task_runner.py:63} INFO - Started process 1366 to run task
[2025-02-03T03:28:46.322+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'csv_dag', 'clear_data_task', 'manual__2025-02-03T03:28:21.469953+00:00', '--job-id', '264', '--raw', '--subdir', 'DAGS_FOLDER/dagto_csv.py', '--cfg-path', '/tmp/tmp_xakx420']
[2025-02-03T03:28:46.324+0000] {standard_task_runner.py:91} INFO - Job 264: Subtask clear_data_task
[2025-02-03T03:28:46.414+0000] {task_command.py:426} INFO - Running <TaskInstance: csv_dag.clear_data_task manual__2025-02-03T03:28:21.469953+00:00 [running]> on host 2892ee60bb8a
[2025-02-03T03:28:46.596+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Andres Urrea' AIRFLOW_CTX_DAG_ID='csv_dag' AIRFLOW_CTX_TASK_ID='clear_data_task' AIRFLOW_CTX_EXECUTION_DATE='2025-02-03T03:28:21.469953+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-02-03T03:28:21.469953+00:00'
[2025-02-03T03:28:46.599+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-02-03T03:28:46.601+0000] {logging_mixin.py:188} INFO - Archivo /tmp/df.json eliminado correctamente.
[2025-02-03T03:28:46.602+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-02-03T03:28:46.603+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-02-03T03:28:46.619+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=csv_dag, task_id=clear_data_task, execution_date=20250203T032821, start_date=20250203T032846, end_date=20250203T032846
[2025-02-03T03:28:46.658+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-02-03T03:28:46.692+0000] {taskinstance.py:3482} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-02-03T03:28:46.696+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
